finetune/base_models/gpt/featurizer.py

```diff
@@ -231,6 +208,8 @@ def add_explain_tokens(X, max_length, pool_idx):
     clf_tok_x_seq_w_pos = tf.concat((clf_tok_x_seq, flat_pos), -1)
     return tf.concat((X, clf_tok_x_seq_w_pos), 1)
 
+def get_pos_values(seq_len, vocab_size):
```

Comment:
This is the change that concerns me the most because of all the little oddities w/ off by one errors, special tokens, mask tokens, etc. that impact where pos embeddings start.  Also the Roberta starting pos embeddings at 2 thing.

Reply:
Roberta already handles it's own, we never used the input idx for roberta, just GPT and GPT2

Reply:
When I printed them out, it seemed to be equivalent to this range call, can you think of any specific cases worth testing.

Reply:
We have tests for this so ultimately not concerned, this was just the only thing that jumped out as having the potential to break something.